split-webapp/split/src/components/TransactionList.js

```diff
@@ -60,66 +20,98 @@ class TransactionList extends React.Component {
 
   componentDidMount() {
     //  actual address http://0.0.0.0:1234/api/bill_data/get_bill
-    fetch("https://jake-good.free.beeceptor.com/my/api/path") // temp
+    fetch("/api/bill_data/get_bills") // temp
       .then(res => {
         return res.json();
       })
       .then(data => {
+        console.log(data.bills);
```

Comment:
Nit: probably don't need to console.log this anymore since it works :)

Reply:
I also made a quick fix to the case you mentioned above - hope that's okay!